sqlite/src/comdb2build.c

```diff
@@ -6602,3 +6632,43 @@ void comdb2_drop_view(Parse *pParse, SrcList *pName)
 out:
     free_schema_change_type(sc);
 }
+
+/* delete entry by seed from comdb2_sc_history 
+ * called from stored procedure function db_comdb_delete_sc_history()
+ */
+int comdb2DeleteFromScHistory(char *tablename, uint64_t seed)
+{
+    BpfuncArg *arg = (BpfuncArg*) malloc(sizeof(BpfuncArg));
```

Comment:
A structure initialized on stack would suffice.

Reply:
yes, i think on the stack works. thanks.